src/measure_theory/measurable_space.lean

```diff
@@ -291,13 +298,17 @@ section constructions
 instance : measurable_space empty := ⊤
 instance : measurable_space punit := ⊤ -- this also works for `unit`
 instance : measurable_space bool := ⊤
+instance Prop.measurable_space : measurable_space Prop := ⊤
+instance {α} : measurable_space (set α) := ⊤
```

Comment:
Why this is the right instance for `measurable_space (set α)`? IMHO, any instance on `set α` that is not equal to the corresponding instance on `α → Prop` needs an explanation in the docstring.

Reply:
I thought I needed this instance but I'm not sure anymore. Let's see whether this PR compiles without it.